types/index.d.ts

```diff
@@ -1331,6 +1331,14 @@ declare namespace RamdaAdjunct {
         padCharsStart(padString: string, targetLength: number, value: string): string;
         padCharsStart(padString: string, targetLength: number): (value: string) => string;
         padCharsStart(padString: string): (targetLength: number, value: string) => string;
+
+        /**
```

Comment:
Please align this description with the final description suggested above: https://github.com/char0n/ramda-adjunct/pull/1287/files#diff-8322d64eab92ea5ea1ca5d559ab14f97R4